### PR TITLE
CSS Bug fix + Add New Networks

### PR DESCRIPTION
**Pull request by ghost**
### Description
Adds the new networks we were missing from ethereum-fork based coins. Also handles the possibility of some dPaths not being provided for unsupported wallets. These changes have not been tested significantly, as I don't have a lot of knowledge about these networks and don't hold any of their tokens.

### Changes
+ Added MUSIC + PIRL+ EAST + ESN network support
+ Add libnotify-bin for travis build

### Screenshots
screenshot

### Live Demo

https://mew.chainkorea.kr

### CODE REVIEW

**Comment by akx20000a** [comment]
@kvhnuke @gamalielhere  Review needed 😄

**Comment by akx20000a** [comment]
@gamelielhere When merge? :sweat_smile:

**Review by eastcoinuser — APPROVED** [review]
ACK

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ env: CXX=g++-4.8
 addons:
   apt:
     sources: ubuntu-toolchain-r-test
-    packages: g++-4.8
+    packages: g++-4.8 libnotify-bin
```
Comment (gamalielhere) [inline .travis.yml:12]: @akx20000a what's this library for? I don't think we need this in production.

Comment (ghost) [reply .travis.yml:12]: @gamalielhere  libnotify-bin is for fixing the gulp-notify error https://laracasts.com/discuss/channels/general-discussion/gulp-notify-error-in-notifier-error-in-plugin-gulp-notify-not-found-notify-send

It helped when I build locally 😄

**Comment by ghost** [comment]
@gamalielhere @kvhnuke  Done for now 😄 

Live demo site: https://mew.chainkorea.kr

File: app/scripts/nodes.js [hunk 2]
```diff
@@ -246,16 +249,81 @@ nodes.nodeList = {
...
+        'service': 'pirl.io',
+        'lib': new nodes.customNode('https://wallrpc.pirl.io', '')
+    },
+    'esn': {
+        'name': 'ESN',
+        'blockExplorerTX': 'https://ethersocial.net/tx/[[txHash]]',
+        'blockExplorerAddr': 'https://ethersocial.net/addr/[[address]]',
+        'type': nodes.nodeTypes.ESN,
+        'eip155': false,
+        'chainId': 1,
```
Comment (kvhnuke) [inline app/scripts/nodes.js:308]: we cannot allow another network type that has the same chainId as the mainnet. This allows generating transactions that can be double spent! 
is this chainId correct?
Edit:
just noticed this doesnt have eip155 enabled as well

File: app/scripts/nodes.js [hunk 3]
```diff
@@ -246,16 +249,81 @@ nodes.nodeList = {
...
+        'service': 'pool.musicoin.tw',
+        'lib': new nodes.customNode('https://mewapi.musicoin.tw', '')
+    },
+    'pirl': {
+        'name': 'PIRL',
+        'blockExplorerTX': 'https://poseidon.pirl.io/explorer/transaction/[[txHash]]',
+        'blockExplorerAddr': 'https://poseidon.pirl.io/explorer/address/[[address]]',
+        'type': nodes.nodeTypes.PIRL,
+        'eip155': true,
+        'chainId': 3125659152,
```
Comment (kvhnuke) [inline app/scripts/nodes.js:295]: are you sure this chainId is valid ?
based on EIP 155 specs chainId is either `v = CHAIN_ID * 2 + 35` or `v = CHAIN_ID * 2 + 36`
Also v is only one byte, unless this network added more bytes to the signature, I believe this chainId is invalid

File: app/scripts/nodes.js (hunk 2 above)
Comment (ghost) [reply app/scripts/nodes.js:308]: @kvhnuke They didn't enabled the eip155 and I don't know why 😄 https://github.com/ethersocial/go-esn/blob/develop/params/config.go#L33

File: app/scripts/nodes.js (hunk 3 above)
Comment (ghost) [reply app/scripts/nodes.js:295]: @kvhnuke You can't believe this but this is true https://github.com/pirl/pirl/blob/master/params/config.go#L33 They enabled the eip155 with 3125659152.

**Comment by akx20000a** [comment]
@kvhnuke If there is a problem on ESN and PIRL node I will delete them immediately.
However, you can test them on https://mew.chainkorea.kr

ps) We need a fast merge on this in order to support our new ETSC network based on new chainId

**Pull request closed without merging**